developer-docs-site/static/examples/rust/first_nft/src/lib.rs

```diff
@@ -0,0 +1,229 @@
+// Copyright (c) Aptos
+// SPDX-License-Identifier: Apache-2.0
+
+use first_transaction::{Account, RestClient};
+use hex;
+use reqwest;
+use serde_json::Value;
+pub struct NftClient {
+    url: String,
+    pub rest_client: RestClient,
+}
+const NUMBER_MAX: u64 = 18446744073709551615;
+impl NftClient {
+    /// Represents an account as well as the private, public key-pair for the Aptos blockchain.
+    pub fn new(url: &str) -> Self {
+        Self {
+            url: url.to_string(),
+            rest_client: RestClient::new(url.to_string()),
+        }
+    }
+    pub fn submit_transaction_helper(&self, account: &mut Account, payload: Value) {
+        let hash = self
+            .rest_client
+            .execution_transaction_with_payload(account, payload);
+        self.rest_client.wait_for_transaction(hash.as_str())
+    }
+    pub fn create_collection(
```

Comment:
we provide this RUST SDK to wrapper around move public script function. It would be better to use the token sdk to replace these function now https://github.com/aptos-labs/aptos-core/blob/main/aptos-move/framework/cached-packages/src/aptos_token_sdk_builder.rs#L129

Reply:
It is amazing! I'll try it